services/net/contentmigration/ContentMigrationAction.cs

```diff
@@ -295,13 +297,46 @@ public class ContentMigrationAction : IngestAction<ContentMigrationOptions>
                 return;
             }
 
-            var sourceContent = contentMigrator.CreateSourceContent(lookups, source!, product, manager.Ingest.IngestType!.ContentType, newsItem, defaultTimeZone);
-
+            var sourceContent = contentMigrator.CreateSourceContent(lookups, source!, product, manager.Ingest.IngestType!.ContentType, newsItem);
+            bool isNewSourceContent = false;
+            bool isUpdatedSourceContent = false;
             var reference = await this.FindContentReferenceAsync(source?.Code, sourceContent.Uid);
             if (reference == null)
             {
-                reference = await this.Api.AddContentReferenceAsync(contentMigrator.CreateContentReference(source!, manager.Ingest.Topic, newsItem, sourceContent.Uid, defaultTimeZone));
+                isNewSourceContent = true;
+                reference = contentMigrator.CreateContentReference(source!, manager.Ingest.Topic, newsItem, sourceContent.Uid);
+            }
+            else
+            {
+                // check if this is content, previously ingested, but has been updated by an Editor in TNO
+                DateTime originalLastUpdateDate = DateTime.MinValue;
+                if (reference.Metadata.ContainsKey(UpdatedOnMetadataKey))
+                {
+                    if (!DateTime.TryParse(reference.Metadata[UpdatedOnMetadataKey].ToString(), out originalLastUpdateDate))
+                        originalLastUpdateDate = DateTime.MinValue;
+                }
+
+                if (sourceContent.UpdatedOn > originalLastUpdateDate)
+                {
+                    isUpdatedSourceContent = true;
+                    reference.Status = (int)WorkflowStatus.Received;
+                    // What about the worst case scenario: one Editor changes it in MMIA and another Editor changes it in TNO?
```

Comment:
This can occur through our automated processes.  Content that is ingested is published separately.  Which results in the `UpdatedOn` being changed.

Reply:
this comparison is between the originalLastUpdateDate from TNO (which we will store with the item upon ingest) and the item currently being ingest from TNO (which we have picked up as an existing reference)